drake/systems/onlinePlanning/SharedDataHandleMutable.m

```diff
@@ -0,0 +1,38 @@
+classdef SharedDataHandleMutable < handle
```

Comment:
Some of the wrapped data gets modified in mex but it looks like SharedDataHandle still works with that case.  I'll zap the Mutable variant.